udata/core/dataset/api_fields.py

```diff
@@ -44,6 +44,9 @@
         required=True, enum=RESOURCE_TYPES.keys()),
     'format': fields.String(description='The resource format', required=True),
     'url': fields.String(description='The resource URL', required=True),
+    'permalink': fields.String(description='The resource permanent URL to '
+                               'the latest version (won\'t change on update)',
```

Comment:
As a consumer of the API, I'd be interested in knowing that I'll have to follow the redirection to actually get the file. OK for the second part.